Add experts map to the repo (#40)
rust-lang/compiler-team: experts/MAP.md

> # Compiler team expert map

Reviewer: ```suggestion
# Expert Map
```

I don't think we need to say "compiler team", since that is implied by this repository.

> This document containts different areas of the compiler and people that

Reviewer: ```suggestion
This document contains the different areas of the compiler and the people that
```

> know a bit about each area. For now that document is on hackmd https://hackmd.io/Izvor8KZRiqUgcUyd2CYTw

Reviewer: ```suggestion
are experts in each area. While the expert map is being fleshed out, [you can find it on HackMD][map].

[map]: https://hackmd.io/Izvor8KZRiqUgcUyd2CYTw
```

# Expert Map

This document contains the different areas of the compiler and the people that

> are experts in each area. While the expert map is being fleshed out, [you can find it on HackMD][map].

Reviewer 2: I would add:

---

This map is intended to help you figure out who you should ask for help if you have questions about how some area works.
